xAPI.md

```diff
@@ -2483,24 +2483,14 @@ See <a href="#AppendixE">Appendix E: Example Signed Statement</a> for an example
 
 ## 3.0 Metadata
 
-### 3.1 IRI Requirements
-
-###### Metadata Requirements
+Metadata is additional information about the resource.  It enables decision making, search, and discoverability  
+within systems.  In xAPI, metadata can be utilized in a variety of locations.  The most common is within [Activity Definitions](#actdef)
```

Comment:
Double space! @andyjohnson you need to start doing a search for "  " in your PRs :-p